Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -1,14 +1,16 @@
 #include "hzpch.h"
+
+// Include GLAD before GLFW, because it otherwise causes a compile error on linux
+#include <glad/glad.h>
```

Comment:
Instead of including it before the header file, why not move it into the header file at once; where GLFW and glad can be included next to each other?